hphp/test/frameworks/frameworks.yaml

```diff
@@ -139,6 +139,13 @@
     commit: 7dc9a43cb856debc22fe158a1144018e16f7a637
     install_root: fastroute
     test_root: fastroute
+  flow:
+    url: https://git.typo3.org/Flow/Distributions/Base.git
+    branch: '3.0'
+    commit: 300f50f02bf41c25677111bd0c8962643b16bff8
+    install_root: flow
+    test_root: flow
```

Comment:
This is incorrect. You need flow/Build/BuildEssentials/PhpUnit

This is why it broke.

Paths in the phpunit xml are interpretted relative to the test root. The paths in yours are relative to the directory containing the phpunit xml.